packages/react-mutation-mapper/src/component/column/Annotation.tsx

```diff
@@ -103,6 +103,8 @@ function getDefaultTumorType(): string {
     return 'Unknown';
 }
 
+const memoized: Map<string, IAnnotation> = new Map();
```

Comment:
Unfortunately, the compiler complains when I add a `@memoize` decorator, probably because the method isn't simple like those in `CosmicColumnFormatter.tsx`